app/src/androidTest/java/org/mozilla/fenix/ui/robots/ThreeDotMenuMainRobot.kt

```diff
@@ -106,6 +109,25 @@ class ThreeDotMenuMainRobot {
     fun verifyShareALinkTitle() = assertShareALinkTitle()
     fun verifyWhatsNewButton() = assertWhatsNewButton()
     fun verifyAddFirefoxHome() = assertAddToFirefoxHome()
+    fun verifyAddToMobileHome() = assertAddToMobileHome()
+    fun verifyDesktopSite() = assertDesktopSite()
+
+    fun verifyThreeDotMainMenuItems() {
```

Comment:
This method will fail in landscape...not all options are visible, you should split it in, at least two, before swiping you see and can verify: 
Settings
Finde in Page
Add to top sites
Add to Home screen
Save to collection
Destop site
And the start, share, reload buttons

Then if you scroll down you see:
Add-ons
History
Bookmarks
Synced tabs

If you try to check all of them at the same time, the test fails

Reply:
okay